From 84d85004983b5c3d5a5ceee00f9e45cfd18a3a23 Mon Sep 17 00:00:00 2001
From: Lim Siew Hoon <siew.hoon.lim@intel.com>
Date: Thu, 5 Jul 2018 20:33:31 +0800
Subject: [PATCH 6/6] To disable hook dist-gzip to dist.

dist-gzip will be added "--best" option which gzip unrecognized
it in MR4 kernel 4.14 Yocto Linux Rootfs build environment, but it
only recognized the "-9". This "--best" option added by auto tools
into Makefile and Makefile. This eventually will causing compress
source code failed lead to build process failed. In configure.ac
the AM_INIT_AUTOMAKE is set to dist-bzip2,but it also hook up dist-gzip
as second option too in compress the source code. Suggesting
to disabled it by add in "no-dist-gzip" to disable dist-gzip hook
to dist since AM_INIT_AUTOMAKE also set to dist-bzip2.

Signed-off-by: Lim Siew Hoon <siew.hoon.lim@intel.com>
Acked-by: Matt Roper <matthew.d.roper@intel.com>
Signed-off-by: Matt Roper <matthew.d.roper@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 3e44805..8c5e941 100644
--- a/configure.ac
+++ b/configure.ac
@@ -37,7 +37,7 @@ XORG_MACROS_VERSION(1.12)
 XORG_WITH_XSLTPROC
 XORG_MANPAGE_SECTIONS
 
-AM_INIT_AUTOMAKE([1.10 foreign dist-bzip2])
+AM_INIT_AUTOMAKE([1.10 foreign dist-bzip2 no-dist-gzip])
 
 # Enable quiet compiles on automake 1.11.
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
-- 
1.9.1

